rules/windows/credential_access_posh_request_ticket.toml

```diff
@@ -14,6 +14,41 @@ index = ["winlogbeat-*", "logs-windows.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "PowerShell Kerberos Ticket Request"
+note = """## Triage and analysis
+
+### Investigating Explicit PowerShell Kerberos Ticket Request
+
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks, making
+it available for use in various environments, creating an attractive way for attackers to execute code.
+
+When an account has an SPN associated, it makes a viable target for a Kerberoasting attack, which consists of brute
+force to crack the user password, which is used to encrypt a Kerberos TGS ticket.
```

Comment:
```suggestion
force to crack the user password and is used to encrypt a Kerberos TGS ticket.
```

Reply:
The last part of this is not clear to me. Do Kerberoasting attacks use the cracked user password to encrypt a TGS ticket? If so I would suggest: "Accounts associated with a service principal name (SPN) are viable targets for Kerberoasting attacks, which use brute force to crack an account's password then use that password to encrypt a Kerberos TGS ticket." 

Otherwise, I'm not sure what "and is used to encrypt a Kerberos TGS ticket" means here.

Reply:
Basically, TGS tickets are encrypted using the user password. If an attacker brute force the ticket, he can retrieve the user password as a result (Success decrypting the ticket == The user password was discovered)